lib/cmd-build.js

```diff
@@ -85,8 +86,13 @@ function build (entry, opts) {
         // on some OSes it is zero-copy all the way; e.g. never leaves the
         // kernel! :D
         function copy (done) {
-          if (fs.copyFile) fs.copyFile(src, dest, fin)
-          else pump(fs.createReadStream(src), fs.createWriteStream(dest), fin)
+          fsCompare.ctime(src, dest, function (err, diff) {
```

Comment:
Should this be modified time instead of created time?

Reply:
Of course—good catch!